molo/surveys/tests/test_templatetags.py

```diff
@@ -73,6 +79,11 @@ def setUp(self):
                 slug="direct_survey_title",
                 display_survey_directly=True,
             )
+        self.client.post(reverse(
```

Comment:
This is the pattern that I've been trying to avoid, because it means that we're testing the templatetag while relying on the view layer. I've been trying to create better test utils more akin to `molo.core` with the `mk_'thing'` pattern and `mk_translation` wrapper.

I'm going to +1 and merge, because we need to unblock other work, but we really need to refactor how we're testing stuff here, and we need to improve the test suite.